debian/entry.sh

```diff
@@ -90,6 +90,8 @@ function init_non_systemd()
 		"$CMD" "$@" &
 		pid=$!
 		wait "$pid"
+		exit_code=$?
+		fg > /dev/null || exit "$exit_code"
```

Comment:
@petrosagg, when '/bin/bash' is started in the background that means it will immediately receive a SIGTSTP on start which puts the process on hold. Then `wait` command will catch this state change and pass on to `fg` which brings the shell back to foreground.